configure.ac

```diff
@@ -227,8 +227,13 @@ AX_CODE_COVERAGE
 AC_ARG_WITH([flux-security], AS_HELP_STRING([--with-flux-security],
              [Build with flux-security]))
 AS_IF([test "x$with_flux_security" = "xyes"], [
-    PKG_CHECK_MODULES([FLUX_SECURITY], [flux-security], [], [])
+    PKG_CHECK_MODULES([FLUX_SECURITY], [flux-security],
+                      [flux_sec_incdir=`$PKG_CONFIG --variable=includedir flux-security`],
+                      [flux_sec_incdir=;])
```

Comment:
Should this line set the value to `x` since that is checked below?

Reply:
I am not sure (I fully admit I'm cargo-cult programming here).  Based on conditionals elsewhere in this autoconf file, it seems that the way to test for an empty string is to check if the string appended to the letter x equals x.  And I believe the semi-colon here terminates the line, meaning `flux_sec_incdir` is an empty string.  Here is the [StackOverflow post](https://stackoverflow.com/questions/21664473/query-pkg-config-variable-through-autotools) that I drew inspiration from.

I could get rid of the `AS_IF` entirely by putting the `AC_MSG_ERROR` in as the last argument to `PKG_CHECK_MODULES` rather than `flux_sec_incdir=;`.

Reply:
Oops, no I think I misread the code - never mind!